_includes/newcomers/newcomers.html

```diff
@@ -0,0 +1,89 @@
+    community.
+</p>
+<p>
+    Also, you can check the quick-links below for jumping stright into things.
+</p>
+<table style="margin:0px; padding:0px;width:100%; table-layout: fixed;">
+    <th>Information</th>
+    <th>Guides</th>
+    <th>Issues up for grabs</th>
+    <tr>
+        <td><a href="" target="_blank">Calender</a> | <a href="" target="_blank">Slack</a> | <a href=""
+                target="_blank">GitHub</a> | <a href="" target="_blank">YouTube</a> | <a href=""
+                target="_blank">Drive</a></td>
+        <td><a href="" target="_blank">First-timers' Welcome Guide</a></td>
+        <td><a href="" target="_blank">help-wanted</a></td>
+    </tr>
+    <tr>
+        <td> </td>
+        <td><a href="" target="_blank">Welcome Newcomers!</a></td>
+        <td><a href="" target="_blank">Layer5 Repository Overview</a></td>
+    </tr>
+    <tr>
+        <td><a href="" target="_blank">Weekly Zoom on Thursdays at 8AM Central</a></td>
+        <td><a href="" target="_blank">Layer5 Repository Overview</a></td>
+        <td><a href="" target="_blank">good first issue</a></td>
+    </tr>
+</table>
+
+<h3>How to contribute?</h3>
+
+<p>
+    The following set of steps outlines the process by which you can openly engage, learn, and participate in the broad
+    set of open source projects at Layer5. If at any time you get stuck, please seek help in the #first-timers channel
+    in the Layer5 (joining instructions below). Our MeshMates and community members are here to help!
+</p>
+
+<h4>Finding an issue to work on</h4>
+
+<ol>
+    <li>Browse the list of <a href="https://github.com/layer5io" target="_blank">Layer5 org</a> issues bearing the “<a
+            href="https://github.com/issues?q=is%3Aopen+is%3Aissue+archived%3Afalse+org%3Alayer5io+label%3A%22good+first+issue%22+"
+            target="_blank">good first issue</a>” label, “<a
+            href="https://github.com/issues?q=is%3Aopen+is%3Aissue+archived%3Afalse+org%3Alayer5io+label%3A%22first+timers+only%22+"
+            target="_blank">first timers only</a>” or the
+        “<a href="https://github.com/issues?utf8=✓&q=is%3Aopen+is%3Aissue+archived%3Afalse+org%3Alayer5io+label%3A%22help+wanted%22+"
+            target="_blank">help wanted</a>” label.</li>
+    <li>Alternatively, find an area of interest by reviewing/browsing the <a
+            href="https://drive.google.com/a/layer5.io/open?id=1brtiJhdzal_O6NBZU_JQXiBff2InNtmgL_G1JgAiZtk"
+            target="_blank">Layer5
+            Repository Overview</a> document.</li>
+    <li>After identifying the issue you wish to work on, check whether it has been assigned or not by taking a look at
+        the assignee section on the issue. If it hasn't been, signal your interest by commenting on the issue so that it
+        can be assigned to you.
+    </li>
+
+</ol>
+
+<h4>Working on your issue</h4>
+
+<p>Once an issue has been assigned to you, it’s time to get started! 👩🏿‍💻</p>
+
```

Comment:
Add a line here which redirects to the CONTRIBUTING.md file. It should be something like `take a look at the <a href = "a
            href="https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md">Contributing Guidelines</a> to contribute to Layer5`

Reply:
Sure, that seems appropriate here.